src/Controller/EventController.php

```diff
@@ -36,6 +35,28 @@ public function listEvent(Request $request, EventRepository $eventRepository)
             ];
     }
 
+    /**
+     * @Route("event/add", name="new_event")
+     * @Template("event/event_add.html.twig")
+     */
+    public function addEvent(Request $request)
+    {
+        $event = new Event;
+        $form = $this->createForm(EventFormType::class, $event);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $event = $form->getData();
+
+            $this->saveEntity($event);
+
+            $this->addNoticeFlash('Event created and is pending validation by administrator"');
```

Comment:
@daria-sieroshtan 
"approval" is the word
Also you should getused to do thing inline, like
```PHP
$form = $this->createForm(EventFormType::class, new Event());
$form->handleRequest($request);
if ($form->isSubmitted() && $form->isValid()) {
    $this->saveEntity($form->getData());
```
Like it or not you **WILL** meet a tonn of code like this

Reply:
@daria-sieroshtan 
please don't ignore me silently; write something like "I don't agree"